src/integration/pages/onDemandTVPage/hausa/__snapshots__/canonical.test.js.snap

```diff
@@ -485,9 +485,9 @@ exports[`Canonical On Demand T V Page SEO Linked data should match text 1`] = `
         "name": "Hausa",
       },
       "mainEntityOfPage": {
-        "@id": "http://localhost:7080/hausa/bbc_hausa_tv/tv/w172xcg0kg6vph8",
+        "@id": "http://localhost:7080/hausa/bbc_hausa_tv/tv/w172yjj7rfhxp1p",
         "@type": "WebPage",
-        "name": "Labaran Talabijin",
+        "name": "Labaran Talabijin - Labaran Talabijin - BBC News Hausa",
```

Comment:
Is this what we would expect?

Reply:
Yes, according to https://validator.schema.org/#url=https%3A%2F%2Fwww.bbc.com%2Fhausa%2Fbbc_hausa_tv%2Ftv%2Fw172yjj7rfhxp1p -> view WebPage. It uses page title + episode title & they are the same